.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "prettier.singleQuote": false
+}
```

Comment:
uh, I'd suggest adding the prettier config instead of updating prettier plugin settings as this will only affect to users who work with VSCode / VSCodium or Cursor, it won't work / affect for users using any other IDEs like Zed, Fleet etc.

We have public [`node`](https://github.com/dscvr-one/prettier-config#node) config for prettier available.

Reply:
I don't think we can add dscvr-one/prettier-config since this is a forked repo. But I agree. I removed this and added `.prettierrc.yml` instead